manifests/0000_08_cluster-dns-operator_03-daemonset.yaml

```diff
@@ -0,0 +1,56 @@
+# Node resolver for internal registry
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: cluster-dns-node-resolver
+  namespace: openshift-cluster-dns-operator
+  labels:
+    openshift-app: dns-node-resolver
+spec:
+  selector:
+    matchLabels:
+      openshift-app: dns-node-resolver
+  template:
+    metadata:
+      labels:
+        openshift-app: dns-node-resolver
+    spec:
+      dnsPolicy: Default
```

Comment:
Doesn't this mean the pod won't actually be using the cluster DNS service to resolve the registry name?